src/reducers/scroller.js

```diff
@@ -32,8 +32,7 @@ function scroller(state = DEFAULT_STATE, action) {
       }
 
       //TODO (asif) reinitalize when viewheight changes
-      var initialState = scrollHelper.initialize(state, props);
-      return Object.assign({}, initialState);
```

Comment:
since your initialize method already does an Object.assign clone, I figured this was redundant and removed it.  Let me know if that seems unsafe